### DISCUSSION TITLE
Does calcDensity of boundary misorientation overemphasize c-axis (quartz)?

### DISCUSSION THREAD

**Discussion by nicolasmroberts** [mtex-toolbox/mtex#568] [Ask Anything]
Hi,

I am using low angle boundary misorientation analysis to interpret slip systems in quartz. I have followed the mtex tutorials (mtex_5.5beta) to calculate 2-10° boundaries, and they give me the same answer as previous versions of mtex.

I am wondering whether density plot of misorientation axes in crystal coordinates is a good way to understand slip system distribution, and whether it might over-emphasize axes close to the 0001 axis of quartz. In the figure below, I show the (1) CPO,  (2) misorientation axes in both pole figure and crystallographic orientation, and (3) misorientation axes that are close (with 15°) to the different major crystallographic axes.

Although the axis distribution in (2) suggest that there are many more axes close to the c-axis than elsewhere, the axis distributions in (3) show that there are many more axes within 15° of a rhomb axis, for example (n= 1993 vs. n = 639). Is this just a reflection that the calc density takes the random misorientation distribution (which favors non-c axes) into account? Or does the smaller physical area in the plot near the c-axis (1/6 of a circle) compared with 1 full circle for rhombs or 1/2 circle for other axes leads to a higher density of c-axis-parallel misorientation axes. 

Thanks for the clarification!

Nick Roberts

<img width="1439" alt="plot axis distrubution question" src="https://user-images.githubusercontent.com/35352527/98563398-beefcc00-2270-11eb-8901-a5da9a8c3c9f.png">

**Comment by zmichels** [comment]
Hi Nick,

When you calculate the density, are you specifying a halfwidth that matches the 15º separation you used for n?

Zach

**Reply by nicolasmroberts** [reply]
Hey Zach,

No -- half-width is 5°. Attached is what it looks like with a 15° halfwidth -- not a big change 
<img width="568" alt="Screen Shot 2020-11-09 at 10 21 27 AM" src="https://user-images.githubusercontent.com/35352527/98567262-57884b00-2275-11eb-8ffb-258867af6c94.png">

**Comment by kilir** [comment]
Hi Nick,
1) if you compare densities (or number of directions) you should probably evaluate the same area or volume, so in your case within a cone of identical width. Also consider that you have many more symmetrically equivalent e.g. rhomb directions than c-axes. calcDensity computes densities as multiples of a uniform distribution. I would say, for axes with small angles, that assumption should be fine.
2) If you want to interpret slip systems based on misorientation axes, you need to assume a) your low angle boundaries are actually subgrain boundaries b) your kinematically important dislocations are those stored in those subgrain boundaries c) you know the character of the boundary (tilt, twist, general).

Hope that helps.
Cheers,
Rüdiger
